public/css/style.css

```diff
@@ -7,10 +7,10 @@
 	--white: #ffffff;
 }
 
-body {
+.column {
 	font-family: "Roboto", sans-serif;
 	font-size: 20px;
-	color: var(--white);
+	color: var(--black);
 	background-color: var(--black);
 }
```

Comment:
Here's why the sides of the page are white and the navbar and other text looks weird: you modified `style.css`. I'm pretty sure this was on accident, but either way, please revert this and take the styles you were looking for into `admin.css`